shared/backend/aws/backend.go

```diff
@@ -182,15 +186,15 @@ func NewAWSBackend(
 
 	userName, err := b.GetUserName(ctx)
 	if err != nil {
-		return nil, errors.Wrapf(err, "unable to retrieve identity info, does aws profile [%s] exist?", *b.awsProfile)
+		return nil, errors.Wrapf(err, "retrieving identity info")
```

Comment:
Why?

Reply:
in some cases, awsProfile is nil and it was causing a panic (the API doesn't use an aws profile)